scripts/deploy.sh

```diff
@@ -48,6 +49,7 @@ if [ -z "${COMPONENT_TARBALL}"  ]; then
     echo "-l|--limit: <SUBSET>   Limit option to pass to ansible (limits hosts)"
     echo "-K|--ask-sudo-pass     Ask for sudo password"
     echo "-n|--no-unarchive      Skip uploading and unarchiving the tarball on the remote"
+    echo "-c|--config-only       Only update the components configuration files (only for AzureMFA-gssp and WebAuthn-gssp)"
```

Comment:
Suggestion: add a list of components that support the -c option and give an error when the -c option is used with a component that does not implement it?

Reply:
Nice idea, I'll try to implement this.